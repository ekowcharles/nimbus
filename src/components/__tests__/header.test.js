import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from '../header'

describe('Header', () => {
  it('renders header with correct title', () => {
    const wrapper = shallow(<Header />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})