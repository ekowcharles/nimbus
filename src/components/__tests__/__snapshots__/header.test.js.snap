// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders header with correct title 1`] = `
<div
  className="area-header"
>
  <img
    alt="nimbus-icon"
    className="nimbus-icon"
    src="icon.png"
  />
   nimbus
</div>
`;
